pokedex-with-bit/app.py

```diff
@@ -20,39 +23,50 @@ def config():
     os.environ['JINA_PORT'] = os.environ.get('JINA_PORT', str(45678))
 
 
-# for index
-def index():
+def index(num_docs: int):
     f = Flow.load_config('flows/index.yml')
+    num_docs = min(num_docs, len(glob(os.path.join(os.getcwd(), IMAGE_SRC), recursive=True)))
```

Comment:
don't think you need to add this `min()`. It's passed as an argument and that should be it

Reply:
Ah I guess it could be possible to have fewer images than that in the folder. Nevermind, good catch